wp-dependency-installer.php

```diff
@@ -225,27 +225,25 @@ public function admin_init() {
 
 				if ( $this->is_active( $slug ) ) {
 					continue;
-				}
-
-				if ( $this->is_installed( $slug ) ) {
+				} elseif ( $this->is_installed( $slug ) ) {
```

Comment:
Is there some reason fro making this an `elseif`?

Reply:
In https://github.com/afragen/wp-dependency-installer/pull/37 i would prefer not to have something like this:

```php
if ( $this->is_active( $slug ) ) {
  if ( $this->is_mandatory_notice( $slug ) ) {
    $this->notices[] = [
      'status'  => 'error',
      /* translators: %s: Plugin name */
      'message' => sprintf( esc_html__( 'The %s plugin is a mandatory plugin.' ), $dependency['name'] ),
      'source'  => $dependency['source'],
    ];
  }
  continue; // <-- hard to see, the "elseif" chain works the same way
}

if ( ! $is_required ) {
  // ...
}
```